admin/sql/create_types.sql

```diff
@@ -3,3 +3,5 @@ CREATE TYPE cf_recording_type AS ENUM('top_artist', 'similar_artist');
 CREATE TYPE mb_missing_data_source_enum AS ENUM('cf', 'artist_map');
 
 CREATE TYPE user_relationship_enum AS ENUM('follow');
+
+CREATE TYPE feedback_type_enum AS ENUM('like_rec', 'love_rec', 'dislike_rec', 'hate_rec', 'bad_rec');
```

Comment:
@alastair these strings can be improved I think!

Reply:
Lets name it recommendation_feedback_type_enum as the other feedback doesnt use this

Reply:
I think that they would be fine without the `_rec` suffix. like, love, dislike, hate, bad_recommendation